cristin-import/src/test/resources/features/DegreePhdRules.feature

```diff
@@ -10,4 +10,9 @@ Feature:
 
   Scenario: Cristin Result "Doctoral dissertation" is converted to an NVA entry grouped by "Degree".
     When the Cristin Result is converted to an NVA Resource
-    Then the NVA Resource has a PublicationContext of type Degree
\ No newline at end of file
+    Then the NVA Resource has a PublicationContext of type Degree
+
+  Scenario: Cristin Result's totalNumberOfPages is copied to NVA publications numberOfPages.
+    Given the Cristin entry has a total number of pages equal to "123"
```

Comment:
Could you make it into a scenario outline and add an example that is not-numerical? e.g., "some pages"? I want it to be obvious that is is not always a number